modin/pandas/test/internals/test_repartition.py

```diff
@@ -46,15 +47,22 @@ def test_repartition(axis, dtype):
 
     if dtype == "DataFrame":
         results = {
-            None: (1, 1),
-            0: (1, 2),
-            1: (2, 1),
+            None: ([4], [3]),
+            0: ([4], [2, 1]),
+            1: ([2, 2], [3]),
         }
     else:
         results = {
-            None: (1, 1),
-            0: (1, 1),
-            1: (2, 1),
+            None: ([4], [1]),
+            0: ([4], [1]),
+            1: ([2, 2], [1]),
```

Comment:
Don't think that these changes are still required. The previous notation with `partitions.shape` was much prettier and easier to understand.

Reply:
ok